### PR TITLE
Instructions added

### PR DESCRIPTION
**Pull request by andysotocruz**

### CODE REVIEW

File: README.md
[the remark this answers is not in the document]
Comment (andysotocruz) [reply README.md:1]: you mean the protection of the global branch of the users right?

File: README.md [hunk 1]
```diff
@@ -1 +1,89 @@
...
+
+## Instructions
+
+1. Clone the repository directly in the terminal.
+
+2. Create a branch out of `develop` from **GitHub**:
+
+    > example: `develop-andres` -> `develop-[name]`.
+
+3. Inside the terminal create a branch based on `develop-[name]`.
```
Comment (eamzea) [inline README.md:20]: that's the correct name?

File: README.md [hunk 2]
```diff
@@ -1 +1,89 @@
...
+
+4. Create folders.
+
+    > example: `andres/[skylne]` -> `[name]-[imageName]`.
```
Comment (eamzea) [inline README.md:24]: Should be only [name]

File: README.md [hunk 3]
```diff
@@ -1 +1,89 @@
...
+
+5. Inside the movie skins create a file.
```
Comment (eamzea) [inline README.md:26]: Name folder

File: README.md [hunk 4]
```diff
@@ -1 +1,89 @@
...
+
+    > example: **index.html**
+
+6. Search for an image and download it.
+
+    > example: `andres/skylne/assets/skylne.png` -> `[name]/[image]/assets/[file]` -> `[name]/[image]/assets/[file]`.
```
Comment (eamzea) [inline README.md:32]: When assets folder was created?

File: README.md [hunk 5]
```diff
@@ -1 +1,89 @@
...
+
+7. Inside the HTML file, make a brief description of what you like most about the [image] .
```
Comment (eamzea) [inline README.md:34]: image*

File: README.md [hunk 6]
```diff
@@ -1 +1,89 @@
...
+
+8. Save, comment and upload changes to GitHub.
+
+9. Download at least 2 more images and in each one a brief description.
+
+    - What would you change about the image
+    - What do you like the least about the image?
+
```
Comment (eamzea) [inline README.md:42]: What?
Review orthography and syntax

File: README.md [hunk 7]
```diff
@@ -1 +1,89 @@
...
+# Ejercicio img Descripcion
+
+1. Clonar el repositorio directamente en la terminal.
+
+2. Crear una rama que salga de `develop` desde **GitHub**:
+
+    > ejemplo: `develop-andres` -> `develop-[nombre]`
+
+3. Dentro de la terminal crear una rama basada en `develop-[nombre]`.
```
Comment (eamzea) [inline README.md:60]: Where is the example?

File: README.md [hunk 8]
```diff
@@ -1 +1,89 @@
...
+
+4. Crear carpetas.
+
+    > ejemplo: `andres/[skylne]` -> `[nombre]-[nombreDeLaImagen]`
```
Comment (eamzea) [inline README.md:64]: Same as the English version

File: README.md [hunk 9]
```diff
@@ -1 +1,89 @@
...
+
+5. Dentro de las caretas de películas crear un archivo.
+
+    > ejemplo: **index.html**
+
+6. Buscar una imagen y descargarla.
+
+    > ejemplo: `andres/skylne/assets/skylne.png` -> `[nombre]/[imagen]/assets/[archivo]`
```
Comment (eamzea) [inline README.md:72]: Same as English version

File: README.md [hunk 10]
```diff
@@ -1 +1,89 @@
...
+
+7. Dentro del archivo HTML, realizar una breve descripción de lo que más les agrade de la [imagen] .
```
Comment (eamzea) [inline README.md:74]: Orthography *
image*

File: README.md [hunk 11]
```diff
@@ -1 +1,89 @@
...
+
+8. Guardar, comentar y subir cambios a GitHub.
+
+9. Bajar al menos 2 imagenes más y en cada una breve descripción.
+
+    - Que cambiarían de la imagen
+    - Que es lo que menos les agrada de la imagen
```
Comment (eamzea) [inline README.md:81]: Orthography and syntax

**Review by eamzea — CHANGES_REQUESTED** [review]
- Read two times before pushing your changes.

- What is the objective of adding 2 more images? Those changes are going to be pushed?

**Pull request closed without merging**